spec/services/previous_course_creation_step_service_spec.rb

```diff
@@ -22,8 +22,12 @@
       include_examples "previous step", :subjects, :level
     end
 
+    context "Current step: Modern languages" do
+      include_examples "previous step", :modern_languages, :subjects
+    end
+
     context "Current step: Age Range" do
-      include_examples "previous step", :age_range, :subjects
+      include_examples "previous step", :age_range, :modern_languages
```

Comment:
This breaks the back link on the age range page if you don't have modern languages selected - as it automatically makes you go forward

Reply:
Fixed & added feature spec